pkg/plugins/golang/v3/api.go

```diff
@@ -107,30 +105,33 @@ After the scaffold is written, api will run make on the project.
 }
 
 func (p *createAPISubcommand) BindFlags(fs *pflag.FlagSet) {
-	fs.BoolVar(&p.runMake, "make", true, "if true, run make after generating files")
-
-	fs.BoolVar(&p.doResource, "resource", true,
+	p.options = &goPlugin.Options{}
+	fs.StringVar(&p.options.Group, "group", "", "resource Group")
+	// TODO: allow the user to specify the domain
+	p.options.Domain = p.config.Domain
```

Comment:
It is not part of the context of this PR. 
```suggestion
```

Reply:
if we would like to add the domain as flag/option then it would need to be in a follow-up. it is very confusing and harder to understand after. could you please remove it?

Reply:
This PR is not adding a new flag. Precisely, it is just adding a `TODO` comment that here is where it should be added in a following PR.

Reply:
Could you please remove it? it is nor part of the context of this PR and IMO leads to a misunderstanding such as that we should do the flag which is something that was NOT discussed and defined at all so far.